### PR TITLE
Use promises and ES2015ify tests

### PR DESCRIPTION
**Pull request by kevva**
Haven't had a chance to test this, but should work.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,40 +3,29 @@ var execFile = require('child_process').execFile;
...
+		return Promise.reject(new Error('Only OS X systems are supported'));
 	}
 
-	fs.readFile('/Library/Preferences/com.apple.Bluetooth.plist', function (err, res) {
-		if (err) {
-			cb(err);
-			return;
-		}
-
+	pify(fs.readFile, Promise)('/Library/Preferences/com.apple.Bluetooth.plist').then(function (res) {
```
Comment (gillstrom) [inline index.js]: `return`

File: index.js [hunk 2]
```diff
@@ -3,40 +3,29 @@ var execFile = require('child_process').execFile;
...
 
-	execFile('./bluetooth', state, {cwd: __dirname}, function (err) {
-		if (err) {
-			cb(err);
-			return;
-		}
-
-		cb();
-	});
+	return pify(execFile, Promise)('./bluetooth', state, {cwd: __dirname});
```
Comment (gillstrom) [inline index.js]: `state` has to be an array I can't get it to work otherwise

**Comment by gillstrom** [comment]
Thanks boy! :earth_africa:

**Comment by kevva** [comment]
:100:

**Pull request merged**